<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CMediaNamingSettings</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CMediaNamingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="treatAsTVShowByDefault">
     <property name="text">
      <string>Treat as TV Show by default? (when auto determine fails)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exactMatchesOnly">
     <property name="text">
      <string>Exact Matches Only?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="onlyTransformDirectories">
     <property name="text">
      <string>Only Transform Directories?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MakeMKV File Format RegEX:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rippedWithMakeMKVRegEX">
       <property name="placeholderText">
        <string>Must include named capture ?&lt;num&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
